packages/@eventual/core-runtime/src/workflow/call-executors-and-factories/transaction-call.ts

```diff
@@ -23,26 +23,14 @@ export function createTransactionWorkflowQueueExecutor(
     new TransactionCallExecutor(transactionClient),
     queueClient,
     (_, result, { executionTime, seq }) => {
-      if (result.succeeded) {
-        return createEvent<TransactionRequestSucceeded>(
-          {
-            type: WorkflowEventType.TransactionRequestSucceeded,
-            result: result.output,
-            seq,
-          },
-          executionTime
-        );
-      } else {
-        return createEvent<TransactionRequestFailed>(
-          {
-            type: WorkflowEventType.TransactionRequestFailed,
-            error: "Transaction Failed",
-            message: "",
-            seq,
-          },
-          executionTime
-        );
-      }
+      return createEvent<TransactionRequestSucceeded>(
+        {
+          type: WorkflowEventType.TransactionRequestSucceeded,
+          result,
+          seq,
+        },
+        executionTime
+      );
```

Comment:
why?

Reply:
The executor returns the value (`any`), not the object with the success boolean. turning `success: false` is done in the nested executor because that logic is shared between the workflow and the normal transaction call now (before it was duplicated).